### ISSUE TITLE
letter 'R' appears at every command prompt

### ISSUE THREAD

**Issue by eric-crowdskout** [warpdotdev/Warp#3969]
### Describe the bug

after last update, all my command lines, after every single command have an 'R' already entered. I have to delete it before entering my next command.

### To reproduce

open a terminal window and see the 'R'

### Operating system

MacOS

### Operating system and version

13.6.1

### Current Warp version

v0.2023.12.05.08.02.stable_00

### Regression

Yes, this bug started recently or with an X Warp version

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
hey @eric-crowdskout Sorry to hear about this blocker. We've noticed some reports about a similar issue with R prints in automatically to the terminal. See this GitHub issue: https://github.com/warpdotdev/Warp/issues/3946

In essence, the problem appears to be caused primarily by unsupported plugins in the shell configuration dotfiles (`~/.zshrc`, `~/.bashrc`, `~/.config/fish/config.fish`).

Some users have reported that `zsh-autocomplete` is the culprit, while others have mentioned it was `znap`, although it may be one of the others we have listed here ( or something completely different ): https://docs.warp.dev/help/known-issues#list-of-incompatible-tools

Once found, you can disable parts of your dotfiles just for Warp by using this conditional statement:
```
# Bash and Zsh
if [[ $TERM\_PROGRAM != "WarpTerminal" ]]; then
# > What you want to disable here <
fi

# Fish
if test "$TERM\_PROGRAM" != "WarpTerminal"
# > What you want to disable here <
end
```
Please let us know if this helps with debugging your issue and what plugin was the culprit so that we can investigate further.

To anyone else facing this issue, please add a :+1: to the original post at the top or comment with your details, and subscribe if you'd like to be notified.

**Comment by Advait-M** [comment]
Hey @eric-crowdskout - thanks for raising this! We've identified the issue and pushed a fix for this! Please update Warp to `v0.2023.12.12.08.02.stable_04` and let us know if you're still experiencing issues!

**Comment by eric-crowdskout** [comment]
it's fixed thanks for fast response!
